### DISCUSSION TITLE
Addon installation count removed?

### DISCUSSION THREAD

**Discussion by jensolafkoch** [statamic/cms#8390] [General]
I always found it very helpful to see the installation count of addons. Seems to be removed - any special reason for that? Maybe because the install from the CP has been removed and delegated to CLI composer commands?

**Comment by jackmcdade** [comment]
It was removed because it was confusing for many people. It showed the number of composer installs, not the number of sites it was installed into. For projects with lots of updates and multiple environments or devs, one site could show hundreds or thousands of installs. People with commercial addons would ask us why their sales didn’t match the installs and we’d have to explain this. It seems better to just not have that number - you can always go see it on packagist.

**Comment by jensolafkoch** [comment]
Thanks, Jack, for the explanation! I'd like kind of a review/star system though, maybe something to think about for the "second party" team?

**Comment by jackmcdade** [comment]
Definitely would like that too! Just a bit more work then swapping a number out. We’ll see!On Jul 1, 2023, at 2:44 AM, Jens Olaf Koch ***@***.***> wrote:﻿
Thanks, Jack, for the explanation! I'd like kind of a review/star system though, maybe something to think about for the "second party" team?
